mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -55,25 +55,20 @@ def _prepare_mrp_move_data_from_stock_move(
             origin = 'po'
             po = move.purchase_line_id.order_id.id
             po_line = move.purchase_line_id.id
-        if move.production_id:
+        elif move.production_id:
             order_number = move.production_id.name
             origin = 'mo'
             mo = move.production_id.id
+        elif move.move_dest_ids:
+            for move_dest_id in move.move_dest_ids:
+                if move_dest_id.production_id:
```

Comment:
just asking: shouldn't this `if` have an `else` now?

Reply:
it has

Reply:
where? 🤔

Reply:
ah, you mean "move_dest_id.production_id" no, it is not needed, it's uses de default origin "mv", that's the point of this fix.

Reply:
ah, ok